CMakeModules/ODMacroUtils.cmake

```diff
@@ -122,6 +122,11 @@ macro( OD_SET_PROG_INSTALL_RPATH )
 
 endmacro( OD_SET_PROG_INSTALL_RPATH )
 
+if ( APPLE )
+    set( CMAKE_INSTALL_RPATH "$<IF:$<CONFIG:Debug>,@loader_path/../../Frameworks/Debug,@loader_path/../Frameworks>" )
```

Comment:
Do not set CMAKE_INSTALL_RPATH
Define: OD_SET_MODULE_INSTALL_RPATH, like OD_SET_PROG_INSTALL_RPATH
Use close to line 417, before:
ADD_TARGET_PROPERTIES( ${OD_MODULE_NAME} )